hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -113,6 +117,18 @@
   private File testDir;
   private StorageContainerManager scm;
 
+  private static final int MAX_LV = HDDSLayoutVersionManager.maxLayoutVersion();
+  private static final LayoutVersionProto LARGER_SLV_LAYOUT_PROTO =
+      toLayoutVersionProto(MAX_LV, MAX_LV + 1);
+  private static final LayoutVersionProto SMALLER_SLV_LAYOUT_PROTO =
+      toLayoutVersionProto(MAX_LV, MAX_LV - 1);
```

Comment:
nit. Just to keep things theoretically correct, can we also set the metadata version to MAX_LV - 1. (Since MLV cannot be > SLV)

Reply:
Done